protected/humhub/modules/admin/models/forms/UserEditForm.php

```diff
@@ -164,7 +164,7 @@ public static function getGroupItems($groups = null)
     public function getAuthClientUserService(): AuthClientUserService
     {
         if ($this->authClientUserService === null) {
-            $this->authClientUserService = new AuthClientUserService($this);
+            $this->authClientUserService = Yii::createObject(AuthClientUserService::class, [$this]);
```

Comment:
@yurabakhtin Do we still need this change?

Reply:
@luke- We don't need, because of the changes https://github.com/humhub/limited-user-admin/issues/5#issuecomment-2688069096.
Reverted [here](https://github.com/humhub/humhub/pull/7445/commits/3f8006e02845c745a20d7b7f260c9db008f18cad).